plugins/ascii.py

```diff
@@ -12,99 +12,3 @@ def train(self, msg):
         msg.reply("._||__|  |  ______   ______   ______ \n"
                   "(        | |      | |      | |      |\n"
                   "/-()---() ~ ()--() ~ ()--() ~ ()--()\n")
-
-    @BotPlugin.command(r'\!whoIsGod')
-    def whoIsGod(self, msg):
-        msg.reply("  _ _____ _ _            \n"
-                  " (_)_   _(_) |_ ___ _  _ \n"
-                  " | | | | | |  _/ _ \ || |\n"
-                  " |_| |_| |_|\__\___/\_,_|\n")
-
-    @BotPlugin.command(r'\!shutup')
-    def shutup(self, msg):
-        msg.reply("┌∩┐(◣_◢)┌∩┐\n")
-
-    @BotPlugin.command(r'\!rose')
-    def rose(self, msg):
-        msg.reply("--------{---(@\n")
-
-    @BotPlugin.command(r'\!happybithday')
-    def happybirthday(self, msg):
-        msg.reply("¸¸♬·¯·♩¸¸♪·¯·♫¸¸Happy Birthday To You¸¸♬·¯·♩¸¸♪·¯·♫¸¸\n")
-
-    @BotPlugin.command(r'\!idonotcare')
-    def idonotcare(self, msg):
-        msg.reply("╭∩╮（︶︿︶）╭∩╮\n")
-
-    @BotPlugin.command(r'\!dice')
-    def dice(self, msg):
-        msg.reply("[::]\n")
-
-    @BotPlugin.command(r'\!spider')
-    def spider(self, msg):
-        msg.reply("/X\('-')/X\\n")
-
-    @BotPlugin.command(r'\!huhu')
-    def huhu(self, msg):
-        msg.reply("█▬█ █▄█ █▬█ █▄█\n")
-
-    @BotPlugin.command(r'\!hugme')
-    def hugme(self, msg):
-        msg.reply("(っ◕‿◕)っ\n")
-
-    @BotPlugin.command(r'\!bunny')
-    def bunny(self, msg):
-        msg.reply("(\__/)\n"
-                  "(O.o )\n"
-                  "(> < )\n")
-
-    @BotPlugin.command(r'\!dragons')
-    def dragons(self, msg):
-        msg.reply(" /\_./o__        __o\._/\\n"
-                  "(/^/(_^^'        '^^_)\^\)\n"
-                  "_.(_.)_            _(._)._.\n")
-
-    @BotPlugin.command(r'\!thisslowos')
-    def thisslowos(self, msg):
-        msg.reply("             _.-;;-._\n"
-                  "      '-..-'|   ||   |\n"
-                  "      '-..-'|_.-;;-._|\n"
-                  "      '-..-'|   ||   |\n"
-                  "jgs   '-..-'|_.-''-._|\n")
-
-    @BotPlugin.command(r'\!beer')
```

Comment:
I think this one would be useful. Take care LechBot strips messages at send time